### ISSUE TITLE
Missing files in FreeVS_Examples.tar.gz

### ISSUE THREAD

**Issue by Koyomi55** [esdolo/FreeVS#24]
Hi, thank you for your great work!

I found that the directory "waymo_pseudoimg_multiframe_example" inside the compressed file "FreeVS_Examples.tar.gz" contains only symbolic links. Would it be possible for you to replace the symbolic links with the actual files? Thank you again.

**Comment by esdolo** [comment]
Sorry for this mistake. I updated the pack and now pseudo-images in the origin trajectory are available.

**Comment by Koyomi55** [comment]
Thank you for your response! 

I have already generated these pseudo-images using the provided scripts and ran your model. The outputs are truly impressive and I learned a lot in the process.

However, while implementing FreeVS, I encountered a few questions that I hope you can clarify:
1. **greyish pseudo images**: The points in the pseudo images appear greyish. I calculated the mean of non-black pixel values in the generated pseudo images and found it to be significantly lower than the mean pixel values of GT images, which confirms my observation. Despite this, the model seems to adapt well to the color shift and produces visually reasonable results. Is there a specific reason for shifting the color of pseudo-images? How does it affect the overall performance?
2. **Training schedule of the released model**: The paper mentioned that the model was trained for 40k iters with bs=8 and video_length=8. However, from the open-sourced code, the default inference setting is video_length=6, and the default training setting is video_length=6, bs=1 and 15k iters. Why is there a mismatch between the paper and the code? I also tested the released model with video_length=16 and got even better results (both visually and of PSNR, SSIM, LPIPS) than video_length=6, which was somewhat unexpected. Would you share some insights on this phenomenon?
3. **Validation scene selection**: What criteria were used to select the validation scenes? The paper states, "We ensure that there is ample space on both sides of the ego vehicle in most frames of the selected sequence to simulate novel trajectories by lateral moving the ego vehicle." However, this criterion seems somewhat loose, as most scenes would likely meet it. Could you elaborate on the selection process?
4. **Challenging scenes**: In some "difficult" scenes (e.g. large background holes, fast moving ego, night) the model produces degraded results. I understand that these limitations are inherent to the input pseudo-images. Could you suggest possible ways to address these issues?

Looking forward to your reply!

**Comment by esdolo** [comment]
1.  The generation process of the pseudo-image does not change the pixel values of the valid points. Is the statistical difference due to the sky area being clipped？
2.  We use FSDP in the training script and set gradient accumulation steps = 4  so the in-fact training batch size = 1\*8 (with 8 GPUS) and training iterations = 15k\*4 = 60k.   I did not evaluate the influence of video_length on image quality metrics. In theory, longer video length can only enhance the consistency of generated videos. Perhaps the better image quality performance comes from the larger training batch of images?
3. I randomly selected scenes from WOD that satisfy: 1. There is ample space on both sides of the ego vehicle, and 2. There are as many dynamic objects as possible. (I considered modeling dynamic objects a strength of FreeVS.) I guarantee that the selection of these sequences was not influenced by the sequence-wise performance of the method. 
4.  To handle background holes and situations of fast-moving ego (essentially problems of insufficient LiDAR observation), we can provide 3D prior based on the estimated depth for where LiDAR sweeps are missing/insufficient. Simply accumulating more temporal observations can also help.  Night scene generation is a difficult problem for video generation and I am afraid we need a better video generation baseline to solve this.

I hope these answers are helpful to you!

**Comment by Koyomi55** [comment]
Thank you for your patient and detailed explanation! I learned a lot.

For the first question, I would like to provide an example.

Below are one of the GT images and pseudo_images from scene 115663:

![Image](https://github.com/user-attachments/assets/0afc08bb-65c5-4282-b78a-522fcae1d95a)

![Image](https://github.com/user-attachments/assets/0fcfdff3-99fe-4efc-ab52-d35319563039)

max, min, mean, var of them are:
GT:         
 `255   0   106.05825647424768   38.57280196161783`
pseudo: (black pixels excluded) 
 `255   1   43.58588178245183     57.88036608072113`

I think the greyishness is not my illusion.

**Comment by Koyomi55** [comment]
I also noticed that you used 5Hz frame data instead of the original 10Hz data of WOD. May I ask what was the consideration behind the choice?

Additionally, which kind of LPIPS did you use? I used vgg and the LPIPS of the results never reached below 0.2. However, the SSIM and PSNR are consistent with the numbers reported in the paper.

**Comment by esdolo** [comment]
About the greyishness problem, maybe it is caused by the lidar sweep distribution in 2D. One thing for sure is that the RGB value of every valid point is not changed.

I used 5Hz frame data simply to save data process/storage cost.  
I used the metric implementation of street gaussians(https://github.com/zju3dv/street_gaussians).

**Comment by Koyomi55** [comment]
Thank you for your helpful answers!

I checked the street gaussian implementation, it's alex vesion of LPIPS.
